Healthy/Classes/Modules/MainTabBarController/MainTabBarController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    
+    private(set) var tabBarAddButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .primary100
+        button.setImage(UIImage(systemName: "plus"), for: .normal)
+        button.tintColor = .white
+        button.layer.cornerRadius = 28
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTabBar()
+        configureVieControllers()
+    }
+
+    private func configureTabBar() {
+        setupMainTabBar()
+        addCenterButton(tabBarAddButton, withSize: CGSize(width: 58, height: 58))
+    }
+
+    private func configureVieControllers() {
+
+        let HomeViewController = UIViewController()
```

Comment:
```suggestion
        let homeViewController = UIViewController()
```

Please take care, variables should never start with a capital letter.